interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpression.scala

```diff
@@ -208,29 +209,18 @@ object SpelExpressionParser extends LazyLogging {
 
   //caching?
   def default(classLoader: ClassLoader,
+              expressionConfig: ExpressionDefinition[_],
               dictRegistry: DictRegistry,
               enableSpelForceCompile: Boolean,
-              strictTypeChecking: Boolean,
-              imports: List[String],
               flavour: Flavour,
-              strictMethodsChecking: Boolean,
-              staticMethodInvocationsChecking: Boolean,
-              typeDefinitionSet: TypeDefinitionSet,
-              methodExecutionForUnknownAllowed: Boolean,
-              dynamicPropertyAccessAllowed: Boolean,
-              spelExpressionExcludeList: SpelExpressionExcludeList,
-              conversionService: ConversionService): SpelExpressionParser = {
+              typeDefinitionSet: TypeDefinitionSet): SpelExpressionParser = {
+
     val parser = new org.springframework.expression.spel.standard.SpelExpressionParser(
       //we have to pass classloader, because default contextClassLoader can be sth different than we expect...
       new SpelParserConfiguration(SpelCompilerMode.IMMEDIATE, classLoader)
     )
-    val propertyAccessors = internal.propertyAccessors.configured()
-
-    val classResolutionStrategy = if (strictTypeChecking) SupertypeClassResolutionStrategy.Intersection else SupertypeClassResolutionStrategy.Union
-    val commonSupertypeFinder = new CommonSupertypeFinder(classResolutionStrategy, strictTypeChecking)
-    val evaluationContextPreparer = new EvaluationContextPreparer(classLoader, imports, propertyAccessors, conversionService, spelExpressionExcludeList)
-    val validator = new SpelExpressionValidator(new Typer(commonSupertypeFinder, new KeysDictTyper(dictRegistry),
-      strictMethodsChecking, staticMethodInvocationsChecking, typeDefinitionSet, evaluationContextPreparer, methodExecutionForUnknownAllowed, dynamicPropertyAccessAllowed))
+    val evaluationContextPreparer = EvaluationContextPreparer.default(classLoader, expressionConfig)
+    val validator = new SpelExpressionValidator(Typer.default(classLoader, expressionConfig, new LabelsDictTyper(dictRegistry), typeDefinitionSet))
```

Comment:
In this usage of `Typer.default` should be `KeysDictTyper` as it was previously

Reply:
fixed